MIGRATION.md

```diff
@@ -3,6 +3,7 @@
 API changes in MoveIt releases
 
 ## ROS Noetic (upcoming changes in master)
+- `CollisionObject` messages are now defined using a `PoseStamped` instead of a `header`. This makes it easier to place objects with multiple shapes and subframes in the scene. Replace `header` with `origin.header` in your Melodic code
```

Comment:
Just making a note that origin.header should be updated to pose.header (per comment here  https://github.com/ros-planning/moveit_msgs/pull/69#discussion_r415990025)